src/components/Components/LinkedSecretsListView/LinkedSecretsListView.tsx

```diff
@@ -0,0 +1,170 @@
+import {
+  COMPONENT_DETAILS_PATH,
+  COMPONENT_LIST_PATH,
+  COMPONENT_LINKED_SECRETS_PATH,
+} from '../../../routes/paths';
+import { RouterParams } from '../../../routes/utils';
+import { ExternalLink, Table } from '../../../shared/components';
+import AppEmptyState from '../../../shared/components/empty-state/AppEmptyState';
+import ErrorEmptyState from '../../../shared/components/empty-state/ErrorEmptyState';
+import FilteredEmptyState from '../../../shared/components/empty-state/FilteredEmptyState';
+import { useNamespace } from '../../../shared/providers/Namespace';
+import { SecretKind } from '../../../types';
+import { useApplicationBreadcrumbs } from '../../../utils/breadcrumb-utils';
+import PageLayout from '../../PageLayout/PageLayout';
+import getListHeader, { SortableHeaders } from './LinkedSecretsListHeader';
+import { LinkedSecretsListRow } from './LinkedSecretsListRow';
+
+const sortPaths: Record<SortableHeaders, string> = {
+  [SortableHeaders.secretName]: 'metadata.name',
+  [SortableHeaders.type]: 'type',
+};
+
+export const LinkedSecretsListView: React.FC = () => {
+  const { componentName, applicationName } = useParams<RouterParams>();
+  const namespace = useNamespace();
+  const applicationBreadcrumbs = useApplicationBreadcrumbs();
+  const [component, componentLoaded, componentError] = useComponent(namespace, componentName);
+  const [activeSortIndex, setActiveSortIndex] = React.useState<number>(SortableHeaders.secretName);
+  const [activeSortDirection, setActiveSortDirection] = React.useState<SortByDirection>(
+    SortByDirection.asc,
+  );
+  const [linkedSecrets, linkedSecretsLoaded, linkedSecretsError] = useLinkedSecrets(
+    namespace,
+    componentName,
+  );
+
+  const LinkedSecretsListHeader = React.useMemo(
+    () =>
+      getListHeader(activeSortIndex, activeSortDirection, (_, index, direction) => {
+        setActiveSortIndex(index);
+        setActiveSortDirection(direction);
+      }),
+    [activeSortDirection, activeSortIndex],
+  );
+
+  const sortedLinkedSecrets = useSortedResources(
+    linkedSecrets,
+    activeSortIndex,
+    activeSortDirection,
+    sortPaths,
+  );
+
+  if (!componentLoaded || !linkedSecretsLoaded) {
+    return (
+      <Bullseye>
+        <Spinner data-test="spinner" />
+      </Bullseye>
+    );
+  }
+
+  if (componentError || linkedSecretsError) {
+    const error = componentError ?? linkedSecretsError;
+    const httpError = HttpError.fromCode(error ? (error as { code: number }).code : 404);
+    return (
+      <ErrorEmptyState
+        data-test="linked-secrets-list-view_error-empty-state"
+        httpError={httpError}
+        title="Unable to load linked secrets"
+        body={httpError?.message.length ? httpError?.message : 'Something went wrong'}
+      />
+    );
+  }
+
+  const EmptyMessage = () => (
+    <FilteredEmptyState
+      onClearFilters={() => {}} // TODO: clear name filter - will be implemented in another ticket
```

Comment:
the filtering logic will be implemented on this ticket -> https://issues.redhat.com/browse/KFLUXUI-419

Reply:
Maybe we just remove this part? let the filter issue owner to handle it?

Reply:
I'll remove it!